### ISSUE TITLE
Optional markdown config

### ISSUE THREAD

**Issue by pnodet** [sxzz/eslint-config#43]
Hi!

Thanks a lot for the hardworking. I'm currently in the process of refactoring our company's shareable config to the new flat implementation and your repository has been really helpful.

While reading the code I noticed that in the `src/presets.ts` file, the `markdown` is imported and then spread inside the `presetLangsExtensions` array.

This array is itself spread into the `basic` array, but then you pass a param to your config as such `markdown: enableMarkdown = true`.

To my current understanding and knowledge, I believe this is currently having no impact since you're either using basic with the markdown or using basic with the markdown twice.

**Comment by sxzz** [comment]
Fixed! Could you please try `@sxzz/eslint-config@3.7.2` and test again?

**Comment by pnodet** [comment]
looks good to me thanks!

**Comment by pnodet** [comment]
If you don't mind me asking, I noticed some packages are wrapped in `interopDefault()` whereas others aren't, how do you know which one to wrap? do you explore the code to see how they are exported for each plugin/package?

I also see that you created a `hasVue()` util in `src/env.ts`
do you think it would be possible to do something like:

 ```diff
- if (enableVue) {
+ if (hasVue()) {
    configs.push(...vue)
  }
```

**Comment by sxzz** [comment]
Good idea. Released as `@sxzz/eslint-config@3.7.3`

**Comment by sxzz** [comment]
For the first question, I just tried them one by one, until all worked well.
